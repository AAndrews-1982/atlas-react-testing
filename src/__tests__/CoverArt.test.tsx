// Test for CoverArt

import { render } from '@testing-library/react';
import { ContextProvider } from "../components/AppContext";
import { expect, test } from 'vitest';
import { CoverArt } from '../components/CoverArt';
import placeholder from "../assets/placeholder.svg";

test("CoverArt renders correctly", () => {
    const {container} = render(
        <ContextProvider>
            <CoverArt />
        </ContextProvider>
    );
    expect(container).toMatchSnapshot();
});
