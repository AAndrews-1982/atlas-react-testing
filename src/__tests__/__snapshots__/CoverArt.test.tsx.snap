// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`CoverArt renders correctly 1`] = `
<div>
  <div
    class="cover-art-container mb-6"
  >
    <img
      alt="Loading Placeholder"
      class="placeholder-img animate-pulse rounded-md"
      src="/@fs/Users/altonandrews/Desktop/atlas-react-testing/src/assets/placeholder.svg"
    />
  </div>
</div>
`;
